docs/Harmonization.md

```diff
@@ -78,8 +79,8 @@ This category of information refers to observations, which detail a technical vu
 
 |attribute|description|impact|
 --- | --- | --- |
-|ddos amplifier|These observations refer to misconfigured network services, which are vulnerable to DDoS reflection often over UDP.|Even if this vulnerability does not directly affect the confidentiality or integrity of the host in question, the resource or upstream bandwidth consumption can affect availability.|
-|vulnerable service|These observations refer to specific technical vulnerabilities present on a network service, which have been assigned a CVE by MITRE.|The CVE assigned to the vulnerability affects the host in various ways. The CIA triad and CVSS score are metrics, which detail the severity of the vulnerability. Remote code execution is a good example of a severe impact.| 
+|cve|These observations help identify specific versions of software or software components which are vulnerable.|These observations lack context of which systems or services are affected by the vulnerability. They can be, however, used to identify attack surface, which is not directly exposed to the Internet (with the help of a software inventory for example).|
```

Comment:
"software or software components"; If we plan to add an option to subscribe to all cve's related to a keyword, then a resulting cve maybe related to hardware (e.g. Meltdown and Spectre)